test/e2e_dra/upgradedowngrade_test.go

```diff
@@ -325,6 +358,28 @@ var _ = ginkgo.Describe("DRA upgrade/downgrade", func() {
 		}).WithTimeout(2*time.Minute).Should(gomega.Succeed(), "regular pod should survive upgrade")
 		tCtx = ktesting.End(tCtx)
 
+		// Clean up admin namespace pods
```

Comment:
FYI @ritazh I added a couple spots to this where the "regular" pods from the admin access tests get cleaned up so each phase of the upgrade/downgrade stays relatively steady-state so the tests I'm adding can verify pods stay unschedulable when all devices have been allocated.